packages/forklift-console-plugin/src/components/mappings/MappingPage.tsx

```diff
@@ -111,6 +111,11 @@ export const commonFieldsMetadataFactory: ResourceFieldFactory = (t) => [
       defaultValues: ['false'],
     },
   },
+  {
+    resourceFieldId: C.READY,
+    label: t('Ready'),
```

Comment:
```suggestion
    label: t('Status'),
```
it's a tricky question, but I prefer being consistent with the providers page over being "correct" here
although we actually show Ready status true/false, the intention here is showing the status of the mappings ...
it will be better to actually have the status, but since we don't, it's the best we can

[this will also affect the content, status is "Ready" and "Not ready" instead of yes/no]